recipes/libzip/all/conanfile.py

```diff
@@ -11,7 +11,7 @@ class LibZipConan(ConanFile):
     license = "BSD-3-Clause"
     topics = ("conan", "zip", "libzip", "zip-archives", "zip-editing")
     exports_sources = ["CMakeLists.txt"]
-    generators = "cmake"
+    generators = "cmake", "cmake_find_package"
```

Comment:
Seems to be ok for most dependencies, except zstd. Upstream relies on a custom FindZstd.cmake, and cmake_find_package will generate Findzstd.cmake

Reply:
Maybe remove conan's `FindZstd.cmake` file to give us more assurance?

Reply:
Not enough, find_package(Zstd) won't find Findzstd.cmake (uppercase vs lowercase).

Moreover, their custom FindZstd.cmake defines a target `Zstd::Zstd`, so upstream CMakeLists links to this target, which doesn't exist in conan cmake_find_package.

Reply:
I meant removing conan's `Findzstd.cmake`.
Or what do you suggest?
Or is the current state ok?

Reply:
Didn't you add `cmake_find_package` to not use custom upstream or built-in Find files, or I missed something?

Reply:
I added it for #5108.
He's seeing missing dl/pthread symbols, used by openssl.

Reply:
Ok I understand, but zstd injection seems to be fragile.

In upstream CMakeLists, I would replace `find_package(Zstd)` by:

```cmake
find_package(zstd REQUIRED)
set(Zstd_FOUND 1)
add_library(Zstd::Zstd INTERFACE IMPORTED)
set_property(TARGET Zstd::Zstd PROPERTY INTERFACE_LINK_LIBRARIES zstd::zstd)
```

Reply:
I'll open a pr at https://github.com/nih-at changing their casing + name of targets.
Then no more patches will be required for future releases.